.github/workflows/run-ci-cd.yaml

```
@@ -103,554 +103,43 @@ jobs:
-      - name: Upload report
-        if: always()
-        uses: actions/upload-artifact@043fb46d1a93c77aae656e7c1c64a875d1fc6a0a  # v7.0.1
-        with:
-          name: zap-baseline-scan-report-${{ github.run_id }}
-          path: report_html.html
-    timeout-minutes: 5
+    uses: ./.github/workflows/run-staging.yaml
+    with:
+      release_version: ${{ needs.set-release-version.outputs.release_version }}
+    secrets:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

actionlint (1.7.12)

[error] secret "AWS_ACCESS_KEY_ID" is not defined in "./.github/workflows/run-staging.yaml" reusable workflow. no secret is defined

(workflow-call)